src/test/unit/pages/GamePage.test.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { GamePage } from '../../../pages/GamePage';
```

Comment:
Created ticket for making this absolute: https://gaming-backlog.atlassian.net/jira/software/projects/GB/boards/1?selectedIssue=GB-37

no need for work on your end for this, just making note of this as it's bothering me :)

Reply:
Ah perfect, thanks!